packages/@okta/vuepress-site/docs/guides/submit-app-prereq/main/index.md

```diff
@@ -95,7 +95,7 @@ The OIN catalog organizes integrations into use cases. From the OIN Manager, you
 
 | Use case | Integration capability |
 | -------- | ---------------------- |
-| [Single Sign-On](https://www.okta.com/integrations/?category=single-sign-on) (most common) | Enables users to access your application from any device with a single entry of their Okta user credentials. This use case is appropriate for Security Assertion Markup Language (SAML) and OpenID Connect (OIDC) integrations. |
+| [Single Sign-On](https://www.okta.com/integrations/?category=single-sign-on) (most common) | Enables users to access your application from any device with a single entry of their Okta user credentials. This use case is appropriate for Security Assertion Markup Language (SAML) and OpenID Connect (OIDC) integrations. <br><br> **Note:** You don't need to select this use case, as `Single Sign-On` isn't an option in the **App use case** dropdown. If your integration supports SSO, `Single Sign-On` is automatically assigned to your integration when it's published. |
```

Comment:
I was thinking that `(most common)` could be changed to `(automatically assigned)` as this category won't be assignable by ISVs